src/app/search/items/item-filter-types.ts

```diff
@@ -31,6 +32,7 @@ export interface FilterContext {
   language: DimLanguage;
   customStats: Settings['customStats'];
   d2Definitions: D2ManifestDefinitions | undefined;
+  ownedItemsInfo: OwnedItemsInfo;
```

Comment:
I don't think we need to add new info to FilterContext since the list of owned items can already be derived from `allItems`.